### PR TITLE
proxy components for angular

### LINKED ISSUE #30: IntelliSense for Angular
### What problem does this feature solve?

No code completion is provided in IDEs if the web components are used in Angular. 

### What does the proposed API look like?

* Generate angular proxies with the stencil compiler
* Update Angular demo app to the latest Angular Version

### PR DESCRIPTION
**Pull request by ibirrer**
* Adds angular proxy-components for the web components using the stencil angular output target. 
* Angular Demo App updated
* Removed six-table component (Breaking Change)

closes #30

### CODE REVIEW

**Review by colinscz — COMMENTED** [review]
It would be great if you could split the commit for the removal of the six-table from the rest of the commits to be clean for the changelog, if we want to generate it from the commits added later on.

**Comment by ibirrer** [comment]
> It would be great if you could split the commit for the removal of the six-table from the rest of the commits to be clean for the changelog, if we want to generate it from the commits added later on.

Yes, I agree. Unfortunately it is not that simple to split, so I hoped it would be enough add the breaking change to the commit message body. See https://github.com/six-group/six-webcomponents/commit/e6d6761e22d5d9e83908fe395620567e57838dd5. 

How about mentioning it in the commit header message as well? Like `feat: add angular proxy components and delete six-table support`

File: examples/angular/.editorconfig
Comment (ibirrer) [inline examples/angular/.editorconfig:1]: generated from angular cli - remove in favor of the .editorconfig in root

File: examples/angular/src/app/input-test/input-test.component.ts
Comment (ibirrer) [inline examples/angular/src/app/input-test/input-test.component.ts:1]: left over from testing -> remove

**Comment by colinscz** [comment]
> > It would be great if you could split the commit for the removal of the six-table from the rest of the commits to be clean for the changelog, if we want to generate it from the commits added later on.
> 
> Yes, I agree. Unfortunately it is not that simple to split, so I hoped it would be enough add the breaking change to the commit message body. See [e6d6761](https://github.com/six-group/six-webcomponents/commit/e6d6761e22d5d9e83908fe395620567e57838dd5).
> 
> How about mentioning it in the commit header message as well? Like `feat: add angular proxy components and delete six-table support`

Yes I like that approach, that works for me, that way it gets mentioned in the history of the changelog.

**Review by colinscz — COMMENTED** [review]
Once the remaining open comments are addressed this could be merged.

**Review by colinscz — APPROVED** [review]

**Pull request merged**